### DISCUSSION TITLE
`column.getValue(row)` but with the right type

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#1701] [Q&A] [answered]
I was surprised that when exploring a CSV sheet, `column.getValue(row)` returns a string instead of the type of the column, which in my case was a float.

Is there a method for getting the value of a column in a row in the native type rather than a string?

**Comment by cool-RR** [comment, marked as answer]
@saulpw told me I should use `getTypedValue`.

**Reply by saulpw** [reply]
Yup!  `getValue` for the "base" value, `getTypedValue` to apply the column type, and `getDisplayValue` to apply the formatting (and always get a string).
